tests/src/lease/repay.rs

```diff
@@ -177,7 +177,7 @@ fn full_repay_with_max_ltd() {
         overdue_interest: LpnCoin::ZERO.into(),
         due_margin: LpnCoin::ZERO.into(),
         due_interest: LpnCoin::ZERO.into(),
-        validity: Timestamp::from_nanos(1537237454879305533),
+        validity: Timestamp::from_nanos(1537237449879305533),
```

Comment:
1. At first sight, I do not see any relation to the other PR changes .... could you explain the need to update this value?
2. Could you add a comment in the code on how this value is calculated? It would be useful in the next code changes.

Reply:
I noticed that this value depends on where the block's update is called (inside Builder::init_profit method).